src/Zynga/Framework/IO/Web/V1/Manager.hh

```diff
@@ -50,4 +53,25 @@ class Manager {
       throw new UnexpectedHttpCodeException((string)$returnCode);
     }
   }
+  
+  public static function makeCurlRequest(
+    UrlBox $uploadUrl,
+    Map<string, mixed> $postParams): array<string, mixed> {
+      if(self::$useMockCurl == false) {
+        $session = curl_init();
+        curl_setopt( $session, CURLOPT_URL, $uploadUrl->get());
```

Comment:
We need to verify the return codes for each of these sets as well.

Another way to do this in a simpler format, use curl_setopt_array

http://php.net/manual/en/function.curl-setopt.php

Reply:
(y)